README.md

```diff
@@ -57,6 +57,12 @@ rtc_build_examples = false
 target_cpu = "x64"
 is_debug = false
 ````
+- If you wish to use the Intel GPRA BWE library, you need to add three additional variables, shown below. The lib root should be the directory that contains `gpra_lib.lib`, and the header root should be directory that contains `grpa_bwe.h`. Using the // syntax, the path starts at the root of your source tree.
```

Comment:
what do you mean by 'the path starts at the root of your SOURCE tree'?  the 'source' refers to?